library/cipher.c

```diff
@@ -849,6 +849,9 @@ static int get_pkcs_padding(unsigned char *input, size_t input_len,
     }
 
```

Comment:
Can you please add a non-regression test (probably an additional check in an existing function)?

Reply:
Done, thanks.

With the robustness fix:
`PASSED (125 suites, 26639 tests run)`

Without the robustness fix:
`FAILED (125 suites, 26639 tests run)`